export/service/ExportPackage.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019  (original work) Open Assessment Technologies SA;
+ *
+ * @author Yuri Filippovich
+ */
+
+namespace oat\taoSyncServer\export\service;
+
+use oat\oatbox\service\ConfigurableService;
+use oat\taoSync\model\dataProvider\SyncDataProviderCollection;
+use oat\taoSync\model\Exception\SyncBaseException;
+use oat\taoSync\package\SyncPackageService;
+use oat\taoSyncServer\export\dataProvider\TestCenter;
+
+class ExportPackage extends ConfigurableService
+{
+    const SERVICE_ID = 'taoSyncServer/exportPackage';
+
+    const FILE_PREFIX = 'syncPackageServer';
+
+    /**
+     * @param int $syncId
+     * @param string $orgId
+     * @throws SyncBaseException
+     */
+    public function createPackage($syncId, $orgId)
+    {
+        $data = $this->getDataProviderCollection()->getData([TestCenter::PARAM_ORG_ID => $orgId]);
+
+        $fileName = self::FILE_PREFIX . '_' . $syncId . '.json';
```

Comment:
Filename like this will not be unique, we can have the same synchronization IDs for different organization IDs. In general I don't think we have to sync ID in package filename. We may have multiple synchronizations when data were not changed but sync ID will be different. In this case we don't have to generate new package each time or even send this package to VM.

Reply:
sync id parameter removed